engine/src/cubos/engine/tools/scene_editor/plugin.cpp

```diff
@@ -0,0 +1,247 @@
+#include <imgui.h>
+
+#include <cubos/core/ecs/query.hpp>
+#include <cubos/core/ecs/commands.hpp>
+
+#include <cubos/engine/tools/scene_editor/plugin.hpp>
+#include <cubos/engine/tools/entity_selector/plugin.hpp>
+#include <cubos/engine/imgui/plugin.hpp>
+
+#include <cubos/engine/assets/plugin.hpp>
+#include <cubos/engine/transform/plugin.hpp>
+#include <cubos/engine/scene/plugin.hpp>
+
+#include <cubos/engine/tools/asset_explorer/plugin.hpp>
+
+#include <cubos/core/settings.hpp>
+
+#include <cubos/engine/transform/position.hpp>
+
+using namespace cubos;
+using namespace cubos::engine::tools;
+using namespace cubos::engine;
+using cubos::core::ecs::Write;
+using cubos::core::ecs::Read;
```

Comment:
Add `using cubos::core::ecs::Commands`, `Settings` and `Entity`, so that the code below is less verbose.